### ISSUE TITLE
Crash when hovering gaze pointer on MeshCollider from AssetBundle

### ISSUE THREAD

**Issue by Softelectro** [microsoft/MixedRealityToolkit-Unity#10233]
## Describe the bug

Hovering the gaze pointer over an object with a MeshCollider component instantiated from an AssetBundle results in an editor crash.

## Additional context

The problem has allready reported and closed in an other thread #7632 but it still have it.

Now the issue is causse by 
```
textureCoord = hitInfo.textureCoord;
lightmapCoord = hitInfo.lightmapCoord;
```

## Some tip that I've found to avoid it

First solution - Export assetbundle with 'Read/Write enable' on mesh importer.
Second solution - Disable mesh collider after instantiation (Don't work perfectly).
Thrid solution - Update 'MixedRealityRaycastHit' script with (Don't know if it breaks the normal way):

```
if (meshCollider == null || meshCollider.sharedMesh.isReadable)
{
    textureCoord = hitInfo.textureCoord;
    textureCoord2 = hitInfo.textureCoord2;
    lightmapCoord = hitInfo.lightmapCoord;
}
else
{
    textureCoord = Vector2.zero;
    textureCoord2 = Vector2.zero;
    lightmapCoord = Vector2.zero;
}
```
if any guy can confirm this solution because this seems the better one. And maybe add it in the next release.

## Your setup

- Unity Version : 2020.3.3f1
- MRTK Version : v2.6.1 et 2.7

## Target platform

- HoloLens 2

Thanks in advance
Softelectro

**Comment by WikkidEdd** [comment]
We hit this same problem and implemented the exact same fix back in January, so no adverse effects ;)

We should have reported it back in January, but other things came up.

**Comment by Softelectro** [comment]
Thanks for your return. I keep this solution and hope microsoft team add it in a next release to avoid forget this modification.

**Comment by keveleigh** [comment]
Thanks for the write-up! Definitely looks like #7632 wasn't enough. I've opened #10370 for our upcoming 2.7.3 release to resolve this.

**Comment by keveleigh** [comment]
This should be fixed via #10370
